tests/Functional/Execution/ExecutionTest.php

```diff
@@ -1462,40 +1474,38 @@ public function testUsesACustomFieldResolver()
         ], $result->toArray());
     }
 
-    /**
-     * @throws \Digia\GraphQL\Error\InvariantException
-     */
-    public function testNonNullFieldQuery()
-    {
-        $schema = newSchema([
-            'query' => newObjectType([
-                'name'   => 'Greeting',
-                'fields' => [
-                    'hello' => [
-                        'type'    => newNonNull(stringType()),
-                        'resolve' => function () {
-                            return null;
-                        }
-                    ]
-                ]
-            ])
-        ]);
-
-        /** @var ExecutionResult $executionResult */
-        $executionResult = graphql($schema, 'query Greeting {hello}');
-
-        $this->assertSame([
-            'errors' => [
-                [
-
-                    'message'   => 'Cannot return null for non-nullable field Greeting.hello.',
-                    'locations' => null,
-                    'path'      => null,
-                ]
-            ],
-            'data'  => null
-        ], $executionResult);
-    }
+//    public function testNonNullFieldQuery()
+//    {
+//        /** @noinspection PhpUnhandledExceptionInspection */
+//        $schema = newSchema([
+//            'query' => newObjectType([
+//                'name'   => 'Greeting',
+//                'fields' => [
+//                    'hello' => [
+//                        'type'    => newNonNull(stringType()),
+//                        'resolve' => function () {
+//                            return null;
+//                        }
+//                    ]
+//                ]
+//            ])
+//        ]);
+//
+//        /** @noinspection PhpUnhandledExceptionInspection */
+//        $executionResult = graphql($schema, 'query Greeting {hello}');
+//
+//        $this->assertSame([
+//            'errors' => [
+//                [
+//
+//                    'message'   => 'Cannot return null for non-nullable field Greeting.hello.',
+//                    'locations' => null,
+//                    'path'      => null,
+//                ]
+//            ],
+//            'data'   => null
+//        ], $executionResult);
+//    }
```

Comment:
What happened to this test?

Reply:
It's redundant if I'm not mistaken.

Reply:
Nevermind, my mistake. I re-added the test case now. Good catch.